fifteen_min/src/viewer.rs

```diff
@@ -269,7 +269,7 @@ fn build_panel(ctx: &mut EventCtx, app: &App, start: &Building, isochrone: &Isoc
 
     rows.push(
         ctx.style()
-            .btn_light_popup_icon_text(
+            .btn_popup_icon_text(
```

Comment:
just a rename here - had a lingering "light" in the button names even though it's used in both themes.